apps/namadillo/src/hooks/useSdk.tsx

```diff
@@ -13,7 +13,21 @@ import {
 } from "react";
 import Proxies from "../../scripts/proxies.json";
 
-export const SdkContext = createContext<Sdk | undefined>(undefined);
+export enum MaspParamsStatus {
```

Comment:
Nitpick, but more an idea. What do you think of using QueryStatus type from `@tanstack/react-query` instead? So we can make this loading state type standard across the app

Reply:
Done!